source/install/install-ubuntu-1404-mattermost.rst

```diff
@@ -0,0 +1,95 @@
+.. _install-ubuntu-1404-mattermost:
+
+Installing Mattermost Server
+============================
+
+Install Mattermost Server on a 64-bit machine.
+
+Assume that the IP address of this server is 10.10.10.2
+
+**To install Mattermost Server on Ubuntu**
+
+1. Log into the server that will host Mattermost Server and open a terminal window.
+
+2. Create a mattermost user and group
+
+    - ``sudo adduser --system --group mattermost``
+
+3. Change to the mattermost home directory.
+
+    ``cd /home/mattermost``
+
+4. Download `the latest version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_ by typing:
+
+   -  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
```

Comment:
The install instructions take place at the command line. That is, you're logged into the server, probably via an SSH session, and probably don't have a GUI available. So if you go to that page on your local machine, it gets downloaded to your local machine, and you still have to upload it to your remote machine.

The way it is in the instructions, you copy/paste the line into your terminal, fix up the URL, hit Enter, and the file is downloaded to the remote server.